scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
I'd be curious as to what the GMP configure was looking for... I need to set up a FreeBSD VM, I guess ;)

The problem is that it might not be just GMP - AFAIU, you didn't run 'ct-ng build-all' on FreeBSD, so this issue may affect other packages as well.